src/nvim/ex_cmds2.c

```diff
@@ -1847,19 +1852,21 @@ void ex_argdelete(exarg_T *eap)
     /* ":1,4argdel": Delete all arguments in the range. */
     if (eap->line2 > ARGCOUNT)
       eap->line2 = ARGCOUNT;
-    int n = eap->line2 - eap->line1 + 1;
```

Comment:
But what is the guarantee that it will be an int? I can open a gigantic file and select all of it for deletion, it is a crazy use case, but shouldn't this be long, or more precisely `linenr_T` instead?